packages/relay-kit/src/packs/safe-4337/Safe4337Pack.ts

```diff
@@ -240,11 +316,35 @@ export class Safe4337Pack extends RelayKitBasePack<{
    * @return {Promise<SafeOperation>} The Promise object will resolve a SafeOperation.
    */
   async createTransaction({
-    transactions
+    transactions,
+    options = {}
```

Comment:
Review JSdoc for this fn. Missing options prop and the style is different from the `getEstimateFee` method for example